rdbsync/__init__.py

```diff
@@ -24,6 +24,7 @@
 POLL_INTERVAL_HELP = ('If provided, the command will run continuously, sleeping for the'
                       ' provided number of seconds after each run.')
 LOG_LEVEL_HELP = 'The log level to use. Options are DEBUG, INFO, WARNING, ERROR, CRITICAL.'
+DRY_RUN_HELP = 'Dry run. If selected, print what would have been added to Fedora ResultsDB.'
```

Comment:
This seems like an unrelated change. If you want to keep it in one commit, please add it to the commit message?